Sources/Testing/Events/Recorder/Event.HumanReadableOutputRecorder.swift

```diff
@@ -171,8 +171,12 @@ extension Test.Case {
   /// - Parameters:
   ///   - includeTypeNames: Whether the qualified type name of each argument's
   ///     runtime type should be included. Defaults to `false`.
+  ///
+  /// - Returns: A string containing the arguments of this test case formatted
+  ///   for presentation, or an empty string if this test cases is
+  ///   non-parameterized.
   fileprivate func labeledArguments(includingQualifiedTypeNames includeTypeNames: Bool = false) -> String {
-    arguments.lazy
+    (arguments ?? []).lazy
```

Comment:
Might be easier to read as something like `guard let arguments else { return ... }`